lib/exqlite/connection.ex

```diff
@@ -439,6 +439,12 @@ defmodule Exqlite.Connection do
 
   defp maybe_changes(_, _), do: nil
 
+  # TODO: should we change the NIF to just return the string
+  defp normalize_columns(columns) do
```

Comment:
@warmwaffles I'm curious your thoughts here. I think we need the extra info for help in decoding the results, but can't find where that lives, and making this change didn't result in any tests failing.

We may need to do this normalization only within the Ecto layer, instead.

Reply:
Yea we can make that change in the nif. It's fairly easy.